### ISSUE TITLE
Helm chart questions

### ISSUE THREAD

**Issue by paolomainardi** [cdr/code-server#914]
Hi everyone and thanks for your hard work guys, the project is amazing.

As in the subject, i am developing an helm chart to contribute the project and i have some questions, let me start with the simplest one:

Do you see any contraindication to mount the persistence storage on `/home/coder` rather than the default path you suggest `/home/coder/project` ?

**Comment by deansheather** [comment]
Thanks for working on a helm chart to contribute to the project! I don't have much experience with Kubernetes or Helm, but is there something wrong with the current [deployment file](https://github.com/cdr/code-server/blob/master/deployment/deployment.yaml) we have included in the repository?

**Comment by paolomainardi** [comment]
Hi Dean! 

Nothing bad with that deployment file but as you can see everything is hardcoded, Ingress, persistence and security missing as well, all the aspect I want to cover with the helm chart, simpler to install and to adapt just by passing user’s custom vaiues.

**Comment by deansheather** [comment]
In that case then, feel free to open a PR replacing the old chart. For the project storage, I'd recommend using `/home/coder/project` for consistency with the Docker image and possibly other deployment methods.

**Comment by paolomainardi** [comment]
Ok thanks

**Comment by sr229** [comment]
I think we @deansheather and @MichaelDesantis has worked already on this so I'll be closing this off.

**Comment by InsOpDe** [comment]
What happened to the `chart`?

**Comment by paolomainardi** [comment]
@InsOpDe it's my question too :)

**Comment by MnrGreg** [comment]
Where does one find the helm chart now?

**Comment by nhooyr** [comment]
No plans for a helm chart at the moment.

Is the docker container not enough?

**Comment by paolomainardi** [comment]
Not enough of course, to deploy it on K8s cluster.
But the question is, why the one I contributed has been removed ?

**Comment by PirateBread** [comment]
Hopefully not because extending this solution to be more useful steps on the toes of the enterprise paid version.

**Comment by nhooyr** [comment]
> Hopefully not because extending this solution to be more useful steps on the toes of the enterprise paid version.

Not at all!

We only removed all these configs (k8s, helm, aws etc.) as no one was maintaining them in favour of a streamlined guide/docs. I think we could add a really simple k8s config but it's so easy to do when you know k8s, you just add a stateful set.

However, given the demand within the issue tracker for these configs, we were obviously wrong and I've created https://github.com/cdr/code-server/issues/2012 and to rectify this.
